src/l10n/jp-JP.js

```diff
@@ -548,7 +553,9 @@ const l10n = {
       heading:
         '簡易トランザクション承認とは、指紋認証や顔認証 ' +
         'を用いて、ADAのトランザクションを行うものです。 ' +
+        ' ' +
```

Comment:
@Leo-Emurgo can we delete this part plz and also the extra line two lines below 🙏